public/js/monitor.js

```diff
@@ -51,15 +51,68 @@ function doOauth(el) {
     return;
   }
 
-  // Preserve entire control function
+  if (!emailWatch) {
```

Comment:
Do we need type-safe checking here? In case someone later sends in `{emailWatch: 0}` ?

Reply:
Added. 

https://github.com/mozilla/blurts-server/blob/1665-add-homepage-fxa-checkbox/public/js/monitor.js#L72-L74